### PR TITLE
Add region_id as a request parameter

### PR DESCRIPTION
**Pull request by SungjunApp**
ticket: https://pixlee.atlassian.net/browse/EN-3557

added region_id as a request parameter to distillery and distillery-analytics to differentiate the analytics events by region.

### CODE REVIEW

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/data/api/BasicAPI.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ interface BasicAPI {
     @GET("api/v2/albums/from_sku")
     fun getPhotosWithSKU(
             @Query("sku") sku: String,
-            @Query("api_key") api_key: String,
+            @Query("api_key") api_key: String?,
```
Comment (jchen123) [inline pixleesdk/src/main/java/com/pixlee/pixleesdk/data/api/BasicAPI.kt:20]: why add question marks here?

**Review by jchen123 — APPROVED** [review]
do we need readme changes?

for ios side do you want to do it or get csaba?

File: app/src/main/java/com/pixlee/pixleeandroidsdk/ui/analytics/KtxAnalyticsFragment.kt [hunk 2]
```diff
@@ -223,4 +270,134 @@ class KtxAnalyticsFragment : BaseFragment() {
...
+    }
+
+    fun readRegionIdFromUI(): Int?{
+        val data = textViewRegionId.text.toString()
+        return if (data.isNotEmpty()) {
+            Integer.valueOf(data)
+        } else null
+    }
+
+    fun readSortOptionsFromUI(): PXLAlbumSortOptions {
```
[the remark this answers is not in the document]
Comment (SungjunApp) [reply app/src/main/java/com/pixlee/pixleeandroidsdk/ui/analytics/KtxAnalyticsFragment.kt:292]: This is to get region id from a text input UI in the analytics demo. To try out different search options when firing analytics calls, I added a feature of search filters to this screen, and region id is a part of the filters.

**Comment by SungjunApp** [comment]
for ios, I'd like to do it.

**Comment by jchen123** [comment]
sounds great!!

**Pull request merged**